src/adaptors/in-memory-articles.ts

```diff
@@ -0,0 +1,33 @@
+import { JsonLdObj, Iri } from 'jsonld/jsonld-spec';
+import { Articles } from './articles';
+
+export default class InMemoryArticles implements Articles {
```

Comment:
might be worth noting somewhere that this is not just an in-memory implementation but also a synchronous one

Reply:
Need tests.

Reply:
Not sure if it factors in, but code that uses `Articles` interface expecting async should be able to use `await` on non promise values (as in not have to check if it is async or not, it just works). Only diff is `await` pushes a micro-task, meaning you can get some slightly unexpected ordering. Demo for the curious: https://codesandbox.io/s/bold-dew-u3iz9